configs/tslint.json

```diff
@@ -1,6 +1,10 @@
 {
   "defaultSeverity": "error",
   "rules": {
+    "file-header": [
+      true,
+      "\n \\* Copyright \\(C) \\d{4} .*\n"
```

Comment:
the build is failing because the pattern is bogus: https://travis-ci.org/theia-ide/theia/jobs/317699219#L711

`)` should be escaped as well, i.e. the pattern should be `"\n \\* Copyright \\(C\\) \\d{4} .*\n"` as posted originally

Please run `yarn` locally to check that linting does not fail. If it fails missing copyrights should be added and bogus fixed.

You can ignore tests failure they are not caused by your changes.

Reply:
Weirdly enough when I escape the second parenthesis it stops working

Reply:
Can you share the error?

Reply:
It just doesn't find the header

Reply:
For all files or just for some? Maybe headers are bogus? If so, It is expected, such headers should be fixed. Can you share the file name?

Reply:
browser.ts (packages/core/src/browser/browser.ts)

https://github.com/theia-ide/theia/blob/master/packages/core/src/browser/browser.ts

Reply:
It should have our copyright as well on the top:
```ts
/*
 * Copyright (C) 2017 TypeFox and others.
 *
 * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 */
```

Reply:
Ok, so adding this copyright on top of the existing one should fix, right. Can i go on and make those changes?

Reply:
yes, a bit more guidance:
- if a copyright is missing then add a new, use `TypeFox` if the first change was done by @kittaakos, @svenefftinge or me otherwise `Ericsson`
- if a copyright exists but bogus then usually spaces are missing before `*`, add missing spaces